erpnext/support/doctype/issue/issue.js

```diff
@@ -74,5 +75,38 @@ frappe.ui.form.on("Issue", {
 				frm.timeline.wrapper.data("split-issue-event-attached", true)
 			}
 		}
-	}
+	},
 });
+
+function set_time_to_resolve_and_response(frm) {
+	const customer = $('div[data-fieldname="customer"]');
```

Comment:
Don't use generic selectors, this can fail in cases where the field is hidden in another page. Use the `frm` api to get the element